dist/assets/css/dashboard.css

```diff
@@ -19028,6 +19028,11 @@ svg {
   border: 0;
 }
 
+.selectize-control.multi {
```

Comment:
fix [sass](https://github.com/tabler/tabler/tree/dev/scss) not css

Reply:
Hey @WinterSilence, thank you for your review. Actually, I added my changes to the SASS file. I run `dist` command (`npm run dist`) to compile SASS files to the CSS files. Shouldn't I compile my changes?